packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -40,178 +39,222 @@ import { API_ERROR_CODES } from 'Constants/api-error-codes';
+                }
+                errors.document_number = isDocumentNumberValid(document_number, document_type);
+            }
+            const required_fields = ['first_name', 'last_name', 'date_of_birth'];
+            const validateValues = validate(errors, values);
+            validateValues(val => val, required_fields, localize('This field is required'));
 
-        return removeEmptyPropertiesFromObject(errors);
-    };
-
-    const submitHandler = async (
-        values: TIdvDocumentSubmitForm,
-        {
-            setSubmitting,
-            setStatus,
-            status,
-        }: FormikHelpers<TIdvDocumentSubmitForm> & FormikState<TIdvDocumentSubmitForm>
-    ) => {
-        setSubmitting(true);
-
-        const request = makeSettingsRequest(values, changeable_fields);
-
-        const data = await WS.setSettings(request);
-
-        if (data?.error) {
-            const response_error =
-                data.error?.code === API_ERROR_CODES.DUPLICATE_ACCOUNT
-                    ? DUPLICATE_ACCOUNT_ERROR_MESSAGE
-                    : GENERIC_ERROR_MESSAGE;
-            setStatus({ ...status, error_message: response_error });
-            setSubmitting(false);
-            return;
-        }
-        const get_settings = await WS.authorized.storage.getSettings();
-        if (get_settings?.error) {
-            setStatus({ ...status, error_message: get_settings?.error?.message ?? GENERIC_ERROR_MESSAGE });
-            setSubmitting(false);
-            return;
-        }
+            if (values.first_name) {
+                errors.first_name = validateName(values.first_name);
+            }
+            if (values.last_name) {
+                errors.last_name = validateName(values.last_name);
+            }
+
+            if (!values.confirmation_checkbox) {
+                errors.confirmation_checkbox = 'error';
+            }
 
-        const submit_data = {
-            identity_verification_document_add: 1,
-            ...formatIDVFormValues(values, selected_country.value),
+            return removeEmptyPropertiesFromObject(errors);
         };
 
-        const idv_update_response = await WS.send(submit_data);
-        if (idv_update_response?.error) {
-            const response_error =
-                idv_update_response.error?.code === API_ERROR_CODES.CLAIMED_DOCUMENT
-                    ? CLAIMED_DOCUMENT_ERROR_MESSAGE
-                    : idv_update_response.error?.message ?? GENERIC_ERROR_MESSAGE;
-            setStatus({ error_msg: response_error });
+        const submitHandler = async (
+            values: TIdvDocumentSubmitForm,
+            {
+                setSubmitting,
+                setStatus,
+                status,
+            }: FormikHelpers<TIdvDocumentSubmitForm> & FormikState<TIdvDocumentSubmitForm>
+        ) => {
+            if (shouldSkipIdv(values?.document_type?.id)) {
+                handleSelectionNext?.(true);
+                return;
+            }
```

Comment:
@yauheni-deriv what happens if the user updates personal details first and then fills "i don't have any of these" in IDV form?

Reply:
there is no api call for this form,  then for this form empty errors object will be returned and no submission happens.
being on manual upload step if user decides to go back, he or she will see old details as they weren’t accepted with next idv steps

Reply:
As discussed lets clarify the expected bahaviour

Reply:
it was decided to implement the api call for updating personal details n separate card